playbook-website/app/views/layouts/application_beta.html.erb

```diff
@@ -4,11 +4,20 @@
     <%= render 'layouts/head'%>
   </head>
   <body class="<%= @body_class %> <%= controller_name %> <%= action_name%>">
-    <%= render 'layouts/mobile_nav'%>
+ 
```

Comment:
Unsure if this is related but this is not desirable  https://share.zight.com/JrummNbA